src/dotty/tools/dotc/util/CommentParsing.scala

```diff
@@ -3,15 +3,10 @@
  * @author Martin Odersky
  * @author Felix Mulder
  */
+package dotty.tools.dotc.util
 
-package dotty.tools
-package dottydoc
-package model
-package comment
-
-import scala.reflect.internal.Chars._
-
-object CommentUtils {
+object CommentParsing {
```

Comment:
Needs a comment what it does.

Also what's the motivation for keeping this in `util`? Is it used outside dottydoc as well?